server/apiserver/argoserver.go

```diff
@@ -376,15 +376,6 @@ func mustRegisterGWHandler(register registerFunc, ctx context.Context, mux *runt
 	}
 }
 
-// Unlike the controller, the server creates object based on the config map at init time, and will not pick-up on
-// changes unless we restart.
-// Instead of opting to re-write the server, instead we'll just listen for any old change and restart.
-func (as *argoServer) restartOnConfigChange(interface{}) error {
```

Comment:
Will this change the existing behavior of argo server?  Argo server will not restart if there is any config change. is it right?

Reply:
Correct. This PR should be marked as a breaking change.